### PR TITLE
added `leaflet hex timeslider` plugin

### PR DESCRIPTION
**Pull request by albertkun**
Would like to contribute my plugin to the Leaflet library.

image

### CODE REVIEW

**Comment by johnd0e** [comment]
@albertkun Have you closed this intentionally?

**Comment by albertkun** [comment]
@johnd0e yeah, as while it is functional, I think I could do a better job of documenting for general use. If you think its worth adding, though, I would be happy to re-open this PR.

**Comment by johnd0e** [comment]
I've reopened it as Draft.
Please tell us when it is ready

**Comment by albertkun** [comment]
@johnd0e will do! thank you!

File: docs/plugins.md [hunk 1]
```diff
@@ -2677,6 +2677,17 @@ Most data is two-dimensional (latitude and longitude), but some data has more di
 			<a href="https://github.com/zimmicz">Michal Zimmermann</a>
 		</td>
 	</tr>
+		<tr>
+		<td>
+			<a href="https://github.com/zimmicz/Leaflet-Timeline-Control">Leaflet Hex Time Slider</a>
```
Comment (Falke-Design) [inline docs/plugins.md:2682]:
```suggestion
			<a href="https://github.com/albertkun/leaflet_hex_timeslider/">Leaflet Hex Time Slider</a>
```

File: docs/plugins.md [hunk 2]
```diff
@@ -2677,6 +2677,17 @@ Most data is two-dimensional (latitude and longitude), but some data has more di
...
+		</td>
+		<td>
+			Minimalistic time slider using leaflet + d3.js and nouislider for displaying time series data using a geoJSON file. <a href="https://albertkun.github.io/leaflet_hex_timeslider/">Demo</a>.
+		</td>
+		<td>
+			<a href="https://github.com/albertkun/leaflet_hex_timeslider/">Albert Kochaphum</a>
```
Comment (Falke-Design) [inline docs/plugins.md:2688]:
```suggestion
			<a href="https://github.com/albertkun/">Albert Kochaphum</a>
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Please change the PR and mark it as "ready for review"

**Comment by Falke-Design** [comment]
We added the plugin in the refactoring where we moved the plugins into seperate files #7805

**Pull request closed without merging**